src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -15,15 +15,40 @@
 
 import React, { useState, ChangeEvent } from 'react';
 import { action } from '@storybook/addon-actions';
+import {
+  InteractionTaskArgs,
+  PublicInteractionTask,
+} from 'storybook-addon-performance';
+import { findByText } from '@testing-library/dom';
+import userEvent from '@testing-library/user-event';
 
 import { Checkbox, CheckboxProps } from './Checkbox';
 import docs from './Checkbox.docs.mdx';
 
+const interactionTasks: PublicInteractionTask[] = [
+  {
+    name: 'Tick checkbox',
+    description: 'Click the checkbox and wait for the label to change',
+    run: async ({ container }: InteractionTaskArgs): Promise<void> => {
+      const checkbox: HTMLElement | null = container.querySelector(
+        'input[type=checkbox]',
+      );
```

Comment:
Note: I switched to `document.querySelector` to select elements in interaction tasks, because @testing-library's `getBy*` breaks when running the tasks on several copies of the component (the query [throws an exception](https://testing-library.com/docs/dom-testing-library/api-queries#queryby) if it finds more than one match). `getAllBy*` also doesn't work because it returns an array of elements.

The [addon's docs](https://github.com/atlassian-labs/storybook-addon-performance#usage-interactions) use `document.querySelector` for this so I think we can do the same